### DISCUSSION TITLE
HEIMAN 	HS2IRC device learn action return error

### DISCUSSION THREAD

**Discussion by xugao-0326** [Koenkk/zigbee2mqtt#11429] [General]
action:create,get_list,delete is ok,but lerarn return fail.
{
  "action": "learn",
  "action_key_code": 1,
  "action_result": "error",
  "battery": 100,
  "devices": [
    {
      "id": 1,
      "key_codes": [],
      "model_type": 55
    }
  ],
  "linkquality": 171
}

**Comment by ndesgranges** [comment]
@zenghuanhuan Have you ever found a solution ?
